lib/filters/organization_filter.rb

```diff
@@ -0,0 +1,19 @@
+module ActiveInteraction
+  class OrganizationFilter < Filter
+    register :organization
+
+    def cast(value, context)
+      schema = OrganizationUpdateSchema.schema[:properties]
+      included_keys = value.keys.map(&:to_sym)
+      if included_keys.all? {|s| schema.key? s } && valid?(value)
+        value
+      else
+        raise InvalidValueError, "#{name}: #{describe(value)}"
+      end
+    end
+
+    def valid?(value)
+      JSON::Validator.validate(OrganizationUpdateSchema.schema, value)
```

Comment:
`fully_validate` returns a hash of errors rather than just true/false. But since JsonSchema's validate just raises an exception if it's not empty (and changing that is out of scope for this PR), I'll just compare it to nil.